### PR TITLE
Add toggle functionality and ui

### LINKED ISSUE #6: Request: Toolbar button on/off
It would be good to have a toolbar button to temporarily on/off this extension.

### PR DESCRIPTION
**Pull request by tcortega**
Hello,

I've been using the Social Ghost extension for a while and made some improvements for my personal use. I thought these changes might be beneficial to other users as well, so I decided to submit this PR for your consideration.

Changes:

1. Added a toggle functionality to enable/disable the extension without uninstalling it.
2. Implemented a user interface with a popup to display the current status and allow toggling.
3. Enhanced debugging capabilities by adding more verbose logging.
4. Updated the manifest to support the new features and ensure compatibility with recent browser versions.
5. Added a generic icon.

I understand that your README suggests opening an issue for major changes. While these changes are significant, I hope you'll consider them as they enhance the user experience without altering the core functionality of the extension.

Please let me know if you'd like any modifications or have any questions about these changes. I'm open to feedback and happy to make adjustments as needed.

### CODE REVIEW

**Comment by nhymxu** [comment]
Hi @tcortega 

Thanks for your contribution. No problem if not have issue.
I'll check your PR ASAP.

But very helpful if you can provide screenshot or short video demo how your code change work in real life.
Base on it, I can make decision quicker.

Thank you!

**Comment by nhymxu** [comment]
Because have issue request this feature before, so I link it to your PR

**Comment by tcortega** [comment]
{21158DD8-68CC-4A94-B89E-108EC903C8CA}
That's the menu, along with the icon while disabled.

Here's how the icon looks when enabled:
{767140AA-AD15-41A6-B279-DEB28A0AB1E2}

**Comment by tcortega** [comment]
@nhymxu There was a bug in the old code, it's now fixed.

File: index.js [hunk 1]
```diff
@@ -12,12 +12,28 @@ if (typeof browser === 'undefined' && !isNodeEnv) {
...
+
+browser.storage.local.get('enabled', function (data) {
+    isEnabled = data.enabled !== false;
+});
+
+browser.storage.onChanged.addListener((changes, area) => {
+    if (area === 'local' && 'enabled' in changes) {
+        isEnabled = changes.enabled.newValue;
+    }
+});
```
Comment (nhymxu) [inline index.js:25]: Everything seems good. But I don't want to break the current user when upgrade extension.

So I think we should make extension on by default. And if `disabled` value → disable extension.

What do you think @tcortega ?

File: index.js [hunk 2]
```diff
@@ -45,4 +63,4 @@ const listenToRequests = () => {
     );
 }
 
-listenToRequests();
+listenToRequests();
```
Comment (nhymxu) [inline index.js:66]: I want file have a blank newline at the end. Can you change it @tcortega ?

**Comment by nhymxu** [comment]
LGTM

I'll implement my comment on another PR

Thank you @tcortega

**Pull request merged**